src/validate-import.ts

```diff
@@ -0,0 +1,26 @@
+import type { Env } from "./env"
+import { normalizeRelativePath } from "./utils"
+import { type Validators, validateId } from "./validate-id"
+
+export function validateImport({
+  id,
+  importValidators,
+  root,
+  importer,
+  env,
+}: {
+  id: string
+  importValidators: Validators
+  root: string
+  importer: string | undefined
+  env: Env
+}): void {
+  validateId({
+    id,
+    env,
+    validators: importValidators,
+    errorMessage: `Import from "${id}"${
```

Comment:
Can we do something like:

```
Error: Import not allowed in client graph
- import:
- imported by:
- rule: 
```

Like the data in a table, but with a quick description up top